### PR TITLE
feat(aim): update screenshots in drop fitlers, remove code styling in…

### PR DESCRIPTION
**Pull request by akristen**
Fast follows for AI monitoring

### CODE REVIEW

**Review by ally-sassman — APPROVED** [review]

File: src/install/ai-monitoring/agent-lang/dotnet-config.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+componentType: default
+headingText: Configure the .NET agent
+freshnessValidatedDate: never
+---
+
+To collect AI data, you need to set certain configurations to enable AI monitoring. We recommend applying these configurations in newrelic.yml, but if you have a more complex set up with multiple environments, you can configure with environment variables.
```
Comment (akristen) [inline src/install/ai-monitoring/agent-lang/dotnet-config.mdx:7]:
```suggestion
To collect AI data, you need to set certain configurations to enable AI monitoring. We recommend applying these configurations in newrelic.ini, but if you have a more complex set up with multiple environments, you can configure with environment variables.
```

File: src/install/ai-monitoring/agent-lang/dotnet-config.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+<CollapserGroup>
+    <Collapser
+        id="config-file"
+        title="Configure through `newrelic.yml`"
```
Comment (akristen) [inline src/install/ai-monitoring/agent-lang/dotnet-config.mdx:12]:
```suggestion
        title="Configure through `newrelic.ini`"
```

File: src/install/ai-monitoring/agent-lang/python-config.mdx [hunk 3]
```diff
@@ -39,7 +39,4 @@ To collect AI data, you need to set certain configurations to enable AI monitori
     </Collapser>
 </CollapserGroup>
 
-* When you disable distributed tracing or enable high security mode, the agent will not capture AI data. 
-* You can refer to the [Python configuration doc](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#ai-monitoring) to learn more about AI monitoring configurations.
-        * Refer to the [event harvest section](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#harvest-limits-span-event-data) for `event_harvest_config.harvest_limits.span_event_data`
-        * Refer to the [custom events section](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#harvest-limits-custom-event-data)  for `event_harvest_config.harvest_limits.custom_event_data`
+To learn about more AI monitoring configurations, see our Customize the agent for [AI monitoring doc](/docs/ai-monitoring/customize-agent-ai-monitoring).
```
Comment (akristen) [inline src/install/ai-monitoring/agent-lang/python-config.mdx:42]:
```suggestion
To learn about more AI monitoring configurations, see [Customize the agent for AI monitoring](/docs/ai-monitoring/customize-agent-ai-monitoring).
```

File: src/install/ai-monitoring/agent-lang/dotnet-config.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        ```
+
+        ```ini
+        NEW_RELIC_SPAN_EVENTS_MAX_SAMPLES_STORED = 10000
+        ```
+
+    </Collapser>
+</CollapserGroup>
+
+To learn about more AI monitoring configurations, see our [Customize the agent for AI monitoring](/docs/ai-monitoring/customize-agent-ai-monitoring) doc.
```
Comment (jeff-colucci) [inline src/install/ai-monitoring/agent-lang/dotnet-config.mdx:63]: I think the 'doc' should be part of the link

**Review by jeff-colucci — APPROVED** [review]
Just one small thing but I think you're good to go either way

**Pull request merged**